values.yaml

```diff
@@ -65,7 +65,12 @@ extraEnvs: []
 logLevel: debug
 
 config: |-
-  # Start with empty config
+  domain: edge_proxy_per_ip
```

Comment:
why you need this config here?
The purpose of initconfig is to start the daemon with a default config.
So i see a good idea to add the config's value in the values.yaml but i don't understand why you set the value "edge_proxy_per_ip" here

Reply:
It's just an example config, so it's not really needed. Changed it back to the previous example config.